webapp/src/main/java/com/box/l10n/mojito/service/ai/translation/AITranslationConfiguration.java

```diff
@@ -0,0 +1,85 @@
+package com.box.l10n.mojito.service.ai.translation;
+
+import com.google.common.collect.Maps;
+import java.time.Duration;
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties("l10n.ai.translation")
+public class AITranslationConfiguration {
+
+  private boolean enabled = false;
+  private int batchSize = 1000;
```

Comment:
Ideally, the system would run smaller jobs, more frequently, vs. larger jobs that take e.g. 1 hour each to execute, as that also gives us faster feedback as to how the work is progressing, the health of the system, earlier failure reporting etc.

What batch size should we pick if we want to have this type of job to run for no longer than around 5 minutes on average? (And what equivalent cron schedule)

Reply:
Smaller jobs also means less disruption and potential work lost during a deployment / machine teardown scenario.

Reply:
The batchSize here is referring to the size of the batch retrieved from the DB and then iterated through as part of the job. The job itself will continue to process batches of entities from the table until it is empty

Reply:
Sounds good! (I'll follow-up separate on tracking job performance)